webpack.config.js

```diff
@@ -20,10 +21,16 @@ const {
  */
 const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 
+const defaultBuildConfig = {
+	entry: {},
+	output: {
+		path: path.resolve( __dirname, 'build' ),
+	},
+};
```

Comment:
During each build step, Webpack creates an empty `dist` directory when `output: {}` is specified. This update prevents that and also addresses code duplication in several places below.